en/core-libraries/internationalization-and-localization.rst

```diff
@@ -632,3 +632,8 @@ option to restrict which locales will automatically be used.
 .. meta::
     :title lang=en: Internationalization & Localization
     :keywords lang=en: internationalization localization,internationalization and localization,language application,gettext,l10n,pot,i18n,translation,languages
+    
+Translate Content/Entities
+=======================================================
```

Comment:
This should be the same length as your heading

Reply:
ohhhhhh, so its not a fixed amount of = but it is realted to the length of the heading... got it